test/test_elan.py

```diff
@@ -1143,42 +1139,26 @@ def test_add_nested_reference_annotations(self):
         self.eaf.add_ref_annotation('orth', 'ref', 0, 'Words here.')
         self.eaf.add_ref_annotation('word', 'orth', 0, 'Words')
 
-
-    def test_to_file_to_eaf(self):
-        x, filepath = tempfile.mkstemp()
-        self.eaf = Eaf('./test/sample_2.8.eaf')
-
-        self.eaf.to_file(filepath)
-
-        with open('./test/EAFv2.8.xsd', 'r') as scheme_in:
-            scheme_root = etree.XML(scheme_in.read())
-        schema = etree.XMLSchema(scheme_root)
-        xmlparser = etree.XMLParser(schema=schema)
-        etree.parse(filepath, xmlparser)
-
-        self.eaf = Eaf('./test/sample_2.7.eaf')
-        self.eaf.to_file(filepath)
-
-        with open('./test/EAFv2.8.xsd', 'r') as scheme_in:
-            scheme_root = etree.XML(scheme_in.read())
-        schema = etree.XMLSchema(scheme_root)
-        xmlparser = etree.XMLParser(schema=schema)
-        etree.parse(filepath, xmlparser)
-
-        self.eaf = Eaf('./test/sample_3.0.eaf')
-        self.eaf.to_file(filepath)
-
-        with open('./test/EAFv3.0.xsd', 'r') as scheme_in:
-            scheme_root = etree.XML(scheme_in.read())
-        schema = etree.XMLSchema(scheme_root)
-        xmlparser = etree.XMLParser(schema=schema)
-        etree.parse(filepath, xmlparser)
-
     def test_parse_eaf(self):
         pass
 
     def test_eaf_from_chat(self):
         pass
 
-if __name__ == '__main__':
-    unittest.main()
+
+@pytest.mark.parametrize(
+    'eaf,schema',
+    [
+        ('sample_2.8.eaf', 'EAFv2.8.xsd'),
+        ('sample_2.7.eaf', 'EAFv2.8.xsd'),
+        #('sample_3.0.eaf', 'EAFv3.0.xsd'),  # FIXME: sample file not there yet
```

Comment:
It seems `sample_3.0.eaf` isn't pushed yet.

Reply:
I've added this